[BUG] java.lang.ExceptionInInitializerError

### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

0.1.0

### Operating System

Linux

### Minecraft Version

1.20.4

### Describe the bug

I am using fabric-language-kotlin-1.10.17+kotlin.1.9.22.jar, It instantly crashes and minecraft launcher says:

```
The game crashed whilst bootstrap
Error: java.lang.ExceptionInInitializerError
```

### Steps to reproduce

Run

### Client Log

```shell
[Info: 2024-01-21 22:12:34.225403188: MinecraftJavaLoggingContext.cpp(65)] Game/FabricLoader/GameProvider (main) Info Loading Minecraft 1.20.4 with Fabric Loader 0.15.6
[Info: 2024-01-21 22:12:34.331731966: MinecraftJavaLoggingContext.cpp(65)] Game/FabricLoader (main) Info Loading 17 mods:
	- fabric-language-kotlin 1.10.17+kotlin.1.9.22
	   |-- org_jetbrains_kotlin_kotlin-reflect 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 1.9.22
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.23.1
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.7.3
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.7.3
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.5.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.6.2
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.6.2
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.6.2
	- fabricloader 0.15.6
	   \-- mixinextras 0.3.2
	- java 17
	- liquidbounce 0.1.0
	- minecraft 1.20.4
[Info: 2024-01-21 22:12:34.380015764: MinecraftJavaLoggingContext.cpp(65)] Game/FabricLoader/Mixin (main) Info SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:<APPDIR>/libraries/net/fabricmc/sponge-mixin/0.12.5+mixin.0.8.5/sponge-mixin-0.12.5+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[Info: 2024-01-21 22:12:34.406533857: MinecraftJavaLoggingContext.cpp(65)] Game/FabricLoader/Mixin (main) Info Compatibility level set to JAVA_17
[Info: 2024-01-21 22:12:34.524836489: MinecraftJavaLoggingContext.cpp(65)] Game/FabricLoader/Mixin (main) Warn Error loading class: com/viaversion/viaversion/protocol/packet/PacketWrapperImpl (java.lang.ClassNotFoundException: com/viaversion/viaversion/protocol/packet/PacketWrapperImpl)
[Info: 2024-01-21 22:12:34.530764092: MinecraftJavaLoggingContext.cpp(65)] Game/FabricLoader/Mixin (main) Warn Error loading class: me/jellysquid/mods/sodium/client/render/chunk/compile/pipeline/BlockOcclusionCache (java.lang.ClassNotFoundException: me/jellysquid/mods/sodium/client/render/chunk/compile/pipeline/BlockOcclusionCache)
[Info: 2024-01-21 22:12:34.653307822: MinecraftJavaLoggingContext.cpp(65)] Game/FabricLoader/MixinExtras|Service (main) Info Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.3.2).
[Info: 2024-01-21 22:12:35.001046672: MinecraftJavaLoggingContext.cpp(65)] Game/FabricLoader/Mixin (main) Info liquidbounce.mixins.json:minecraft.entity.MixinPlayerEntityAccessor from mod liquidbounce->@Accessor[FIELD_GETTER]::getTrackedMainArm()Lnet/minecraft/class_2940; should be static as its target is
[Info: 2024-01-21 22:12:36.198908884: MinecraftJavaLoggingContext.cpp(65)] Game/com.mojang.datafixers.DataFixerBuilder (Datafixer Bootstrap) Info 198 Datafixer optimizations took 74 milliseconds
[Info: 2024-01-21 22:12:36.896254164: MinecraftJavaLoggingContext.cpp(65)] Game/net.minecraft.class_6396 (main) Warn Failed to get system info for GL Caps
java.lang.IllegalStateException: Rendersystem called from wrong thread
	at com.mojang.blaze3d.systems.RenderSystem.constructThreadException(RenderSystem.java:185)
	at com.mojang.blaze3d.systems.RenderSystem.assertOnRenderThread(RenderSystem.java:174)
	at com.mojang.blaze3d.systems.RenderSystem.getCapsString(RenderSystem.java:874)
	at net.minecraft.class_6396.method_37123(class_6396.java:66)
	at net.minecraft.class_310.method_37274(class_310.java:2609)
	at net.minecraft.class_310.method_22681(class_310.java:2580)
	at net.minecraft.client.main.Main.main(Main.java:178)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

[Info: 2024-01-21 22:12:36.978330205: MinecraftJavaLoggingContext.cpp(65)] Launcher/launcher (main) Info 
---- Minecraft Crash Report ----
// Ooh. Shiny.

Time: 2024-01-21 23:12:36
Description: Bootstrap

java.lang.ExceptionInInitializerError
	at net.ccbluex.liquidbounce.features.itemgroup.ClientItemGroups.<clinit>(ClientItemGroups.kt:51)
	at net.minecraft.class_7923.handler$zcm000$liquidbounce$injectInitializeTabs(class_7923.java:534)
	at net.minecraft.class_7923.method_47476(class_7923.java:240)
	at net.minecraft.class_2966.method_12851(class_2966.java:69)
	at net.minecraft.client.main.Main.main(Main.java:170)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.NullPointerException
	at net.ccbluex.liquidbounce.config.ConfigSystem.<clinit>(ConfigSystem.kt:275)
	... 8 more
```


### Screenshots

_No response_

--- a/src/main/kotlin/net/ccbluex/liquidbounce/LiquidBounce.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/LiquidBounce.kt
@@ -135,7 +135,7 @@ object LiquidBounce : Listenable {
             InventoryTracker
             WorldToScreen
             Reconnect
-            ClientItemGroups
+            ConfigSystem.root(ClientItemGroups)
             Chat
             BrowserManager
             Fonts

--- a/src/main/kotlin/net/ccbluex/liquidbounce/features/itemgroup/ClientItemGroups.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/features/itemgroup/ClientItemGroups.kt
@@ -47,10 +47,6 @@ object ClientItemGroups : Configurable("tabs") {
     private var beenSetup = false
     val containers by textArray("Containers", mutableListOf())
 
-    init {
-        ConfigSystem.root(this)
-    }
-
     fun storeAsContainerItem(compound: NbtCompound) {
         val compoundString = compound.toString()
 

